package.json

```diff
@@ -33,5 +33,9 @@
     "hooks": {
       "pre-commit": "pretty-quick --staged"
     }
+  },
```

Comment:
I believe prettier and react are already dependencies in `frontend/package.json` so you shouldn't need this here